glusterd2/transaction/transaction.go

```diff
@@ -146,6 +151,18 @@ func (t *Txn) Do() error {
 	return nil
 }
 
+func isNodeUnreachable(err error) bool {
+	unreachable := true
+	if s, ok := err.(*stepResp); ok {
+		for _, e := range s.Resps {
```

Comment:
Just am observation:

Each step response will contain responses from multiple nodes. It is possible that, one among such nodes is down (unreachable error) and there's a genuine error on one of the other nodes. 

This "checkalive and ignore error" infra is only used by volume status command. In case of volume status, all or any error from one or more nodes shouldn't stop the transaction, not just node unreachable error. So we should continue running txn steps regardless of type of error.

For now, this solution is okay, for the purposes of volume status.

Reply:
if anyone of the error messages is not equal to unreachable error, we are returning the error message